### PR TITLE
fix: links in the special shout outs

### PR DESCRIPTION
**Pull request by dshukertjr**
Fixes the mistake in the link of the contributors.

### CODE REVIEW

File: apps/www/_blog/2022-10-20-supabase-js-v2-released.mdx [hunk 1]
```diff
@@ -35,7 +35,7 @@ If you're one of those contributors, thank you!
 - [`storage-js`](https://github.com/supabase/storage-js/graphs/contributors) (17)
 - [`supabase-js`](https://github.com/supabase/supabase-js/graphs/contributors) (39)
 
-Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@bnjmnt4n](https://github.com/pixtron), and [@karlseguin](https://github.com/karlseguin).
+Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@bnjmnt4n](https://github.com/Acknowledgements), and [@karlseguin](https://github.com/karlseguin).
```
Comment (stavares843) [inline apps/www/_blog/2022-10-20-supabase-js-v2-released.mdx:38]: should be https://github.com/bnjmnt4n instead? https://github.com/Acknowledgements is a empty profile

Comment (dshukertjr) [reply apps/www/_blog/2022-10-20-supabase-js-v2-released.mdx:38]: Wow, all the mistakes that I make. How did I do that😂
```suggestion
Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@pixtron](https://github.com/Acknowledgements), and [@karlseguin](https://github.com/karlseguin).
```

File: apps/www/_blog/2022-10-20-supabase-js-v2-released.mdx [hunk 2]
```diff
@@ -35,7 +35,7 @@ If you're one of those contributors, thank you!
 - [`storage-js`](https://github.com/supabase/storage-js/graphs/contributors) (17)
 - [`supabase-js`](https://github.com/supabase/supabase-js/graphs/contributors) (39)
 
-Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@bnjmnt4n](https://github.com/pixtron), and [@karlseguin](https://github.com/karlseguin).
+Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@pixtron](https://github.com/Acknowledgements), and [@karlseguin](https://github.com/karlseguin).
```
[the remark this answers is not in the document]
Comment (pixtron) [reply apps/www/_blog/2022-10-20-supabase-js-v2-released.mdx:38]: Guess it should be 

```md
Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@bnjmnt4n](https://github.com/bnjmnt4n) and [@karlseguin](https://github.com/karlseguin).
```

[the remark this answers is not in the document]
Comment (stavares843) [reply apps/www/_blog/2022-10-20-supabase-js-v2-released.mdx:38]: sorry for tagging you 😂

[the remark this answers is not in the document]
Comment (dshukertjr) [reply apps/www/_blog/2022-10-20-supabase-js-v2-released.mdx:38]:
```suggestion
Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@pixtron](https://github.com/pixtron), and [@karlseguin](https://github.com/karlseguin).
```

File: apps/www/_blog/2022-10-20-supabase-js-v2-released.mdx [hunk 3]
```diff
@@ -35,7 +35,7 @@ If you're one of those contributors, thank you!
 - [`storage-js`](https://github.com/supabase/storage-js/graphs/contributors) (17)
 - [`supabase-js`](https://github.com/supabase/supabase-js/graphs/contributors) (39)
 
-Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@bnjmnt4n](https://github.com/pixtron), and [@karlseguin](https://github.com/karlseguin).
+Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@pixtron](https://github.com/pixtron), and [@karlseguin](https://github.com/karlseguin).
```
Comment (dshukertjr) [inline apps/www/_blog/2022-10-20-supabase-js-v2-released.mdx:38]:
```suggestion
Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@ bnjmnt4n](https://github.com/bnjmnt4n), and [@karlseguin](https://github.com/karlseguin).
```

File: apps/www/_blog/2022-10-20-supabase-js-v2-released.mdx [hunk 4]
```diff
@@ -35,7 +35,7 @@ If you're one of those contributors, thank you!
 - [`storage-js`](https://github.com/supabase/storage-js/graphs/contributors) (17)
 - [`supabase-js`](https://github.com/supabase/supabase-js/graphs/contributors) (39)
 
-Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@bnjmnt4n](https://github.com/pixtron), and [@karlseguin](https://github.com/karlseguin).
+Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@ bnjmnt4n](https://github.com/bnjmnt4n), and [@karlseguin](https://github.com/karlseguin).
```
Comment (dshukertjr) [inline apps/www/_blog/2022-10-20-supabase-js-v2-released.mdx:38]:
```suggestion
Special shout outs to: [@vejja](https://github.com/vejja), [@pixtron](https://github.com/pixtron), [@bnjmnt4n](https://github.com/bnjmnt4n), and [@karlseguin](https://github.com/karlseguin).
```

**Review by kiwicopple — APPROVED** [review]

**Pull request merged**